README.md

```diff
@@ -82,6 +82,7 @@ gatk|4.beta.6|docker pull pgc-images.sbgenomics.com/d3b-bixu/gatk:4.beta.6
 gatk|4.beta.6-tabix|docker pull pgc-images.sbgenomics.com/d3b-bixu/gatk:4.beta.6-tabix
 gatk|latest|docker pull pgc-images.sbgenomics.com/d3b-bixu/gatk:latest
 gistic|latest|docker pull pgc-images.sbgenomics.com/d3b-bixu/gistic:latest
+hbc_scrna_qc|v1.0.0-scrnaseq|docker pull pgc-images.sbgenomics.com/d3b-bixu/hbc_scrna_qc:v1.0.0-scrnaseq
```

Comment:
I think you should just go with version `scrnaseq` even though that's dumb.

Reply:
I thought deviating from semantic versioning would break the GHA. Maybe I over thought that.